.storybook/main.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     '@storybook/addon-a11y',
     '@storybook/addon-links',
     '@storybook/addon-interactions',
+    '@storybook/addon-storysource',
```

Comment:
For the order of the panels, how would you feel about:

- Controls
- Story
- Interactions
- Accessibility
- Actions
- Performance

Also, do you think with the StorySource being added that removing the `@whitespace/storybook-addon-html` would be appropriate or is that serving a different use-case?

Reply:
Oh I love this! I didn't think about changing the order, let me give that a go.

The `html` addon shows the source `html` markup which can be helpful for accessibility testing (or just quickly getting feedback about dom structure as you push changes). However, I think its broken in development 😦 it looked like it had a hate around it to only run in dev mode, but its not running at all for me.